samples/openthread/coap_client/README.rst

```diff
@@ -131,6 +138,9 @@ To activate the optional extensions supported by this sample, modify :makevar:`O
 
 * For the Minimal Thread Device variant, set :file:`overlay-mtd.conf`.
 * For the Multiprotocol Bluetooth LE extension, set :file:`overlay-multiprotocol_ble.conf`.
+* For the Device Firmware Upgrade using SMP extension, set :file:`overlay-dfu_support.conf`.
```

Comment:
Extend Twister configuration for CoAP client, by adding new test case for `overlay-dfu_support.conf` in `sample.yaml` please.

Reply:
Done

Reply:
Yes, unfortunately it isn't yet. That is because we are missing in nRF Connect SDK toolchain a way to build output update files for both application and network core in case of using multiprotocol image. AFAIK it should work when using hci_rpmsg image, however I didn't test it yet, as it wasn't really useful case for me. I believe there already was PR (https://github.com/nrfconnect/sdk-nrf/pull/4529) that should solve it, however it stuck a little bit due to partition manager changes